src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -967,6 +981,20 @@ static grpc_lb_policy *glb_create(grpc_exec_ctx *exec_ctx,
   glb_policy->args = grpc_channel_args_copy_and_add_and_remove(
       args->args, args_to_remove, GPR_ARRAY_SIZE(args_to_remove), &new_arg, 1);
 
+  /* Fallback is off */
+  glb_policy->fallback = false;
+
+  /* Copy the backend server addresses from the resolver for fallback */
+  grpc_lb_addresses *backends = grpc_lb_addresses_create(
+      addresses->num_addresses - num_grpclb_addrs, NULL);
+  for (size_t addr_copied = 0, i = 0; i < addresses->num_addresses; ++i) {
+    if (!addresses->addresses[i].is_balancer) {
+      memcpy(backends->addresses + addr_copied++, addresses->addresses + i, 
```

Comment:
(It would be `&backends->addresses[addr_copied++]` and `&addresses->addresses[i]`.)

Because that will be logically indirect to get the parameter for `memcpy`. It's like I want to mail a gift to my friend, and I go to her house to ask her address to fill the form. I think the difference is very slight and it's more like a personal preference. I can change it if it is preferred in our team.